### ISSUE TITLE
[Question] The performance for Hopper-v3 doesn't get converged for PPO

### ISSUE THREAD

**Issue by cx441000319** [DLR-RM/rl-baselines3-zoo#376]
### ❓ Question

Hi there,

I ran into such an issue when I trained an agent using PPO in Hopper-v3. Here is the performance for 5 seeds by running: **python3 scripts/all_plots.py -a ppo --env Hopper-v3 -f logs/downloaded**

![image](https://user-images.githubusercontent.com/105121907/235363938-0072246f-ea81-4970-83c3-97eafce4fa00.png)

The commands are like: **python train.py --algo ppo --env Hopper-v3 --seed 500X**;

The seeds are from 5000 to 5004 and the default hyper-parameters are used. It always converges to 1K quickly, dramatically decreases to under 100, and then converges to 1K, ... .

I only encountered such an issue for Hopper-v3 (A2C suffers as well). It works well for other environments.

Is there anything I did wrong? Any help is appreciated!

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/DLR-RM/rl-baselines3-zoo/issues) in the repo
- [X] I have read the [SB3 documentation](https://stable-baselines3.readthedocs.io/en/master/)
- [X] I have read the [RL Zoo documentation](https://rl-baselines3-zoo.readthedocs.io)
- [X] If code there is, it is minimal and working
- [X] If code there is, it is formatted using the [markdown code blocks](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) for both code and stack traces.

**Comment by cx441000319** [comment]
Sorry, what more information is needed?

**Comment by qgallouedec** [comment]
This is a fairly common result of ppo, here's a thread about many others that discusses it:

https://www.reddit.com/r/reinforcementlearning/comments/bqh01v/having_trouble_with_ppo_rewards_crashing/

You can try to decrease the cliping parameter and early stopping the experiment.

**Comment by cx441000319** [comment]
Thank you for your suggestions. I will try them.

Based on the experiment results I got, there are another two spot points:
1. PPO works well for Walker2d-v3 and HalfCheetah-v3 (by working well I mean I can generate comparable performance as the benchmark with the default hyper-parameters)
2. A2C also fails in Hopper-v3 as below:

![image](https://user-images.githubusercontent.com/105121907/235373600-a048c6a4-f97a-4bb2-844f-558de8883744.png)

In this case, can I regard it as an issue in dealing with Hopper-v3 instead of an issue in PPO?

**Comment by araffin** [comment]
> Sorry, what more information is needed?

The hyperparameters used and your system/lib information (os, gym version, mujoco version, sb3 version, ...)
